bin/generate-readme.js

```diff
@@ -132,6 +132,30 @@ async function getRepos () {
   while (url) {
     const res = await github.request({ url: url.href });
     repos.push(...res.data.items.map(r => r.full_name));
+    if (res.incompleteResults === true) {
```

Comment:
So, I think I did a bad job of explaining what's up here.  What we've learned recently is that the Search API can never be trusted to return complete results :/  As a result, it's a bad fit for cases where complete results are a requirement.  

I think we specifically need to query this endpoint, using pagination, to build a full list of available repositories:
https://docs.github.com/en/rest/reference/repos#list-organization-repositories

It's also possible I'm misreading the "incomplete results" docs